### ISSUE TITLE
Does the package support GO v1.17.2

### ISSUE THREAD

**Issue by gitHusband** [tidwall/buntdb#84]
I can not download the package using
`go get -u github.com/tidwall/buntdb`

The error is
`go get: github.com/tidwall/buntdb@v1.2.9 requires

        github.com/tidwall/assert@v0.1.0: stream error: stream ID 7; INTERNAL_ERROR`

My Go Version:

`go version go1.17.2 darwin/amd64`

**Comment by tidwall** [comment]
Yes this package does support Go v1.17.2. The error you show looks a lot like a networking issue.

**Comment by gitHusband** [comment]
@tidwall Yeah, thanks! I can get it
